docs/provisioners/helm.md

```diff
@@ -27,8 +26,7 @@ all present the same content, a directory containing a helm chart, in a differen
 The `helm` provisioner can install and make available a specific `helm+v3` bundle in the cluster.
 
 Simply create a `BundleDeployment` resource that contains the desired specification of a Bundle resource.
-The `helm` provisioner will retrieve the referred helm chart and instantiate the Bundle onto the cluster, and eventually install the helm chart
- on the cluster.
+The `helm` provisioner will retrieve the referred helm chart and instantiate a Bundle for it onto the cluster to eventually install.
```

Comment:
nit: I think the original language was a bit easier to read.

> The helm provisioner will retrieve the configured helm chart, instantiate a Bundle onto the cluster, and eventually install the desired helm chart on the cluster.

Might be a middle ground here. WDYT?

Reply:
Yep, that works for me!